setup.cfg

```diff
@@ -41,8 +41,8 @@ setup_requires = setuptools_scm
 
 [options.extras_require]
 tests =
-  dkist-data-simulator
-  dkist-inventory
+  dkist-data-simulator@git+https://bitbucket.org/dkistdc/dkist-data-simulator@visp_wcs
```

Comment:
Make sure to update these once all PRs are merged

Reply:
I think these might have to stay until those branches are released. This is a real sticky circular dependency issue.